pkgs/top-level/all-packages.nix

```
@@ -27039,6 +27039,8 @@ in
 
   write_stylus = libsForQt5.callPackage ../applications/graphics/write_stylus { };
 
+  wlc = with python3Packages; toPythonApplication wlc;
```

I would make this specific to an application
```suggestion
  wlc = callPackage  ../tools/misc/wlc { };
```